tasks/deprecated-image-check.yaml

```diff
@@ -0,0 +1,72 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: deprecated-image-check
+spec:
+  stepTemplate:
+    volumeMounts:
+      - name: data
+        mountPath: /data
+
+  params:
+    - name: registry
+      description: "Image registry for the tested image"
+      default: registry.access.redhat.com
+    - name: source-image
+      description: "Image name to test"
+    - name: policy-dir
+      description: "Path to the directory containing conftest policies"
+      default: "/project/image/policy/deprecated-image.rego"
+
+  results:
+    - name: pyxis-http-code
+      description: HTTP code returned by Pyxis API endpoint
+
+  steps:
+    # Download Pyxis metadata about the image
+    #
+    # project
+    # └── policy
+    # data
+    # └── image_data.json
+    - name: query-pyxis
+      image: registry.access.redhat.com/ubi8/ubi
+      script: |
+        #!/usr/bin/env bash
```

Comment:
I know it's completely legal to embed script logic within a tekton task but I wonder if we should encapsulate all this pyxis logic into a lightweight script in our hacbs-test repo. That way we can make the changes against the script rather than against the task. Keeps the tasks cleaner and less change since it's always interfacing with the same entry point.   Thoughts?